sgpt/app.py

```diff
@@ -51,6 +51,13 @@ def main(  # pylint: disable=too-many-arguments,too-many-locals
         help="Generate and execute shell commands.",
         rich_help_panel="Assistance Options",
     ),
+    describe_shell: bool = typer.Option(
```

Comment:
Could you please add [test_integration](https://github.com/TheR1D/shell_gpt/blob/main/tests/test_integration.py) for this option?